code_challenges.rb

```diff
@@ -2,28 +2,68 @@
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
 
-
-
-
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
 
 
-# --------------------2) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data from the bike object.
-
+def odd_least(arr)
+    arr.select! { |el| el.is_a? Numeric }
+    arr.select(&:odd?).sort
+end
```

Comment:
cool! this can even be one line 
`arr.select! { |el| el.is_a? Numeric }.select(&:odd?).sort`